__tests__/endpoint.test.js

```diff
@@ -3,6 +3,8 @@ const testData = require("../db/data/test-data");
 const seed = require("../db/seeds/seed");
 const request = require("supertest");
 const connection = require("../db/connection.js");
+const expectedResponse = require("../endpoints.json");
```

Comment:
A more descriptive name that indicates what this 'expectedResponse' is meant to be would be better.